### ISSUE TITLE
Exception On First Run With Multiple Displays

### ISSUE THREAD

**Issue by zachfedor** [minbrowser/min#170]
Just downloaded Min-v1.2.2-darwin-x64.zip from the releases page, unzipped, moved to Applications, and opened the app when I see this:
```
Uncaught Exception:
TypeError: Cannot read property 'getPrimaryDisplay' of null
    at createWindow (/Applications/Min.app/Contents/Resources/app/main.build.js:21:27)
    at EventEmitter.functionName (/Applications/Min.app/Contents/Resources/app/main.build.js:138:3)
    at emitTwo (events.js:92:20)
    at EventEmitter.emit (events.js:172:7)
```
I'm running OS X version 10.11.4 (15E65) on a MBP with an external monitor.

**Comment by PalmerAL** [comment]
It appears that OS X is triggering the ```activate``` event before the app is ready for some reason, which means that ```electron.screen``` hasn't loaded yet.

According to the Electron docs, this event "usually happens when the user clicks on the application’s dock icon", but I guess it can happen at other times as well.

**Comment by PalmerAL** [comment]
I think this is fixed in 267e90a113daf0a2accee0721c0659a6951df476. Can you try building Min from the master branch and make sure this is fixed?

**Comment by zachfedor** [comment]
This is fixed. Built using the instructions for devs in the project readme and ran without issue.

Thanks @PalmerAL!
